EKF/common.h

```diff
@@ -208,7 +208,7 @@ struct parameters {
 
 	// process noise
 	float gyro_bias_p_noise{1.0e-3f};	// process noise for IMU rate gyro bias prediction (rad/sec**2)
-	float accel_bias_p_noise{3.0e-3f};	// process noise for IMU accelerometer bias prediction (m/sec**3)
+	float accel_bias_p_noise{6.0e-3f};	// process noise for IMU accelerometer bias prediction (m/sec**3)
```

Comment:
This is going to be hard to achieve for all setups because the fusion mode, observation rates and observation noise parameters used all affect how rapidly the variances reduce. The value of 0.06 is still too small for some setups, but we cannot make it large without having noisy delta velocity bias estimates for others.